### PR TITLE
[Added] info link and description to account and export

### PR DESCRIPTION
**Pull request by matthiashanel**
Currently each value is limited to 255 char to limit excesses

Signed-off-by: Matthias Hanel <mh@synadia.com>

### CODE REVIEW

**Comment by aricart** [comment]
make fmt - needs to run the failed builds are about formatting.

File: v2/types.go [hunk 1]
```diff
@@ -19,10 +19,36 @@ import (
 	"encoding/json"
 	"fmt"
 	"net"
+	"net/url"
 	"strings"
 	"time"
 )
 
+const MaxInfoLength = 255
```
Comment (aricart) [inline v2/types.go:27]: ha! this is a Pascal string? 255 sounds reasonable.

**Review by aricart — APPROVED** [review]
LGTM! - can you run `make fmt` to fix the formatting issue?

File: v2/types.go (hunk 1 above)
Comment (matthiashanel) [reply v2/types.go:27]: I just wanted to keep the description at a reasonable length.  We can increase that any time.

**Comment by aricart** [comment]
It is fine

**Pull request merged**